packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -144,3 +144,16 @@ governing permissions and limitations under the License.
     margin-left: calc(var(--spectrum-global-dimension-size-150) * -1);
   }
 }
+
+
+@media (forced-colors: active) {
+  .spectrum-Popover {
+    /* With a popover that contains a menu,
+    the menu scrolls. Add overflow: hidden,
+    so the menu's scrollbar doesn't overflow the
+    the popover's border-radius in the corners. */
+    &:has(> [class*='spectrum-Menu-popover']) {
```

Comment:
We can't have `overflow: hidden` globally on the popover, because it would hide the `.spectrum-Popover-tip`.

Reply:
`:has()` is supported in Chrome and Edge, which matter most for WHCM. I can use `:has(> [role="menu"], > [role="listbox"])` instead of the class name.